### PR TITLE
Added English language for readme and interface and German language for interface

### PR DESCRIPTION
**Pull request by smoochy**
Hi,

thanks for this integration.

I am a Hydrao user myself. I have applied for an API Key from Hydrao, but did not receive it yet. Once I have it, I will test your integration.

I am not familiar with the French language, so I created an English translation for the readme and the interface, and a German translation for the interface in addition. I am not sure if the German Umlauts are causing any issues. If necessary, I can replace them. Just let me know.

Cheers,
smoo

### CODE REVIEW

File: custom_components/hydrao/translations/us.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "email": "User name (normally email address)",
+                    "password": "Password",
+                    "api_key": "API Key"
+                },
+                "description": "Enter your Hydrao login credentials",
+                "title": "Authentification"
```
Comment (kamaradclimber) [inline custom_components/hydrao/translations/us.json]: I would suggest to use "Authentication"

**Review by kamaradclimber — APPROVED** [review]

File: custom_components/hydrao/translations/us.json (hunk 1 above)
Comment (smoochy) [reply custom_components/hydrao/translations/us.json]: Yes, makes absolutely sense. Thanks for spotting. Will fix it.

**Review by kamaradclimber — APPROVED** [review]

**Review by sebcaps — APPROVED** [review]
LGTM.
Thx for the update.
One disclaimer anyway, this integration almost works : I didn't manage to avoid 429 error from hydrao (too many requests)even increasing update internal. In addition my shower head did not work anymore (no more light) and thus not using the integration. 
Maybe you'll have more luck/ competencies than me 😉

**Comment by smoochy** [comment]
Thank you. 

I had my shower head replaced already 2 times because it stopped working. 

I'm still waiting for the API Key. How long did it take you to get a reply from their support?

//Edit: would you maybe edit the links I put into both Readme files to not link to the /blob/ paths, but to the normal paths? I was not able to do this on my local fork.

**Pull request merged**